.github/workflows/tests_linux_x86_mpi.yml

```diff
@@ -157,6 +157,113 @@ jobs:
           rm -rf * .git .gitignore .github
           pip cache purge
 
+
+  python_tests:
+    if: contains(github.event.pull_request.labels.*.name, 'ci:use-multi-gpu-runner') || (inputs.lightning-version != '' && inputs.pennylane-version != '')
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - ubuntu-22.04
+      - multi-gpu
+    strategy:
+      max-parallel: 1
+      matrix:
+        mpilib: ["mpich", "openmpi"]
+    timeout-minutes: 30
+
+    steps:
+      - name: Checkout pennyLane-lightning-gpu
```

Comment:
```suggestion
      - name: Checkout pennylane-lightning
```

Reply:
changed this in `tests_linux_x86_mpi_gpu.yml`

Reply:
changed this in `tests_linux_x86_mpi_gpu.yml`